src/cysignals/implementation.c

```diff
@@ -21,6 +22,23 @@ Interrupt and signal handling for Cython
  * along with cysignals.  If not, see <http://www.gnu.org/licenses/>.
  *
  ****************************************************************************/
+#define ENABLE_DEBUG_CYSIGNALS 0
+#if ENABLE_DEBUG_CYSIGNALS
+#define DEBUG(...) fprintf(stderr, __VA_ARGS__); fflush(stderr);
```

Comment:
Don't do this. Use `cysigs.debug_level` instead.

Reply:
If i understand correctly the different `debug_level`s are here to indicate how much details we want to be printed rather than a trace category like `WARNING`, `INFO` `DEBUG` ?

Reply:
> rather than a trace category like WARNING, INFO DEBUG ?

It's all debug information. It's more like `DEBUG_A_LITTLE`, `DEBUG_SOME_MORE` and `DEBUG_A_LOT`.